test/index.js

```diff
@@ -54,18 +50,11 @@ describe('Vero', function(){
   describe('mapper', function(){
     describe('identify', function(){
       it('should map basic identify', function(){
-        test.set({ authToken: 'some-auth-token' });
+        test.set({ authToken: settings.authToken });
```

Comment:
Is there a point to incorporating this into the mapper tests? If (when) the auth token changes we're going to have to update all fixtures. Seems like unnecessary coupling?